### DISCUSSION TITLE
paddle_serving部署教程中，模型转换代码实效。

### DISCUSSION THREAD

**Discussion by greatliu** [PaddlePaddle/PaddleOCR#13514] [Q&A]
#### 问题描述 / Problem Description
paddle_serving部署教程中，下载完推理模型，进行转换的时候，因为ocr更新至2.8,paddple更新至2.6,转换脚本失效。
paddle的fluid时效，替换成base后，仍有很多地方失效，导致教程不能进行下去。
有py3.9能用的最新的转换脚本吗？

#### 运行环境 / Runtime Environment
- OS:ubuntu22.04
- Paddle:2.6.0 gpu
- PaddleOCR:2.8.0

#### 复现代码 / Reproduction Code
# 转换检测模型
python3 -m paddle_serving_client.convert --dirname ./ch_PP-OCRv3_det_infer/ \
                                         --model_filename inference.pdmodel          \
                                         --params_filename inference.pdiparams       \
                                         --serving_server ./ppocr_det_v3_serving/ \
                                         --serving_client ./ppocr_det_v3_client/

# 转换识别模型
python3 -m paddle_serving_client.convert --dirname ./ch_PP-OCRv3_rec_infer/ \
                                         --model_filename inference.pdmodel          \
                                         --params_filename inference.pdiparams       \
                                         --serving_server ./ppocr_rec_v3_serving/  \
                                         --serving_client ./ppocr_rec_v3_client/

**Comment by GreatV** [comment]
paddle serve已经不维护了，换其他的部署吧

**Comment by greatliu** [comment]
> paddle serve已经不维护了，换其他的部署吧

你的用户名……跟我差不多。
我第一次做这个部署，我的需求是把表格识别部署到linux上，然后用户在客户端发送请求后，得到的识别结果支持二次处理，比如识别出的内容需要去空格。
我这种需求，用什么部署合适？了解的少，清指教。

**Reply by SWHL** [reply]
建议去问问GPT吧
